camel-cics/src/test/java/com/redhat/camel/component/cics/AbstractCICSTest.java

```diff
@@ -0,0 +1,100 @@
+package com.redhat.camel.component.cics;
+
+import org.apache.camel.Exchange;
+import org.apache.camel.builder.RouteBuilder;
+import org.apache.camel.component.mock.MockEndpoint;
+import org.apache.camel.test.junit5.CamelTestSupport;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.utility.MountableFile;
+
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.redhat.camel.component.cics.CICSConstants.CICS_COMM_AREA_SIZE_HEADER;
+import static com.redhat.camel.component.cics.CICSConstants.CICS_DEFAULT_ENCODING;
+import static com.redhat.camel.component.cics.CICSConstants.CICS_PROGRAM_NAME_HEADER;
+import static com.redhat.camel.component.cics.CICSConstants.CICS_RC_HEADER;
+import static org.apache.camel.ExchangePattern.InOut;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public abstract class AbstractCICSTest extends CamelTestSupport {
+
+    public static final Logger LOG = LoggerFactory.getLogger(AbstractCICSTest.class);
+
+    protected final static Network network = Network.newNetwork();
+    /** CTG Server */
+    protected static final String CTG_SERVER = "cics_server_name";
+
+    protected static final GenericContainer<?> ctgContainer = new GenericContainer<>("images.paas.redhat.com/fuseqe/ibm-cicstg-container-linux-x86-trial:9.3")
```

Comment:
the version used here is 9.3, while the ctg client lib is 9.2, is it ok?

Reply:
We don't have choice, there's no 9.2 container, IBM started doing it from 9.3